### PR TITLE
DBZ-543 Removing a few unused methods

### PR DESCRIPTION
**Pull request by gunnarmorling**
https://issues.jboss.org/browse/DBZ-543

### CODE REVIEW

**Comment by gunnarmorling** [comment]
@jpechane A quick one I noticed while reviewing #396 :)

File: debezium-core/src/test/java/io/debezium/data/KeyValueStore.java [hunk 1]
```diff
@@ -122,10 +122,6 @@ public Collection getOrCreate(String fullyQualifiedName) {
         return getOrCreate(TableId.parse(fullyQualifiedName));
```
Comment (jpechane) [inline debezium-core/src/test/java/io/debezium/data/KeyValueStore.java]: Is this method used?

**Comment by jpechane** [comment]
@gunnarmorling Good sight :-). I think I am fine, just maybe one more method can be removed

**Comment by gunnarmorling** [comment]
@jpechane, ah, yes. Pushed an update.

**Comment by jpechane** [comment]
Merged, thanks!

**Pull request merged**